app/views/approval_items/_items.html.erb

```diff
@@ -8,16 +8,26 @@
 
 <div class="flex flex-col section-gap-default mt-2">
   <%= render V2::CardComponent.new(title: "Items", separator: false, size: :full) do |card| %>
-    <% card.with_action do %>
-      <%= render V2::ButtonComponent.new(
-        scheme: :light,
-        options: root_path,
-        type: :link,
-        disabled: true,
-        size: :xxs,
-        label: "Copy from previous release") do |b|
-        b.with_icon("v2/square_stack.svg", size: :sm)
-      end %>
+    <% if release.approvals_editable? %>
```

Comment:
**suggestion:** I think we can move this button to the left of the `Add` button, that will avoid doing `approvals_editable?` check twice in the same page.